boards/hamilton/dist/flash.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
I think this file is probably not needed: you can reuse the script in `dist/tools/jlink` by setting JLink specific variable in including `jlink.inc.mk` in the Makefile.include of that board.

Reply:
This comment also stands for reset.sh and debug.sh

Reply:
done